cypress/support/step_definition/common_helper/prePostConditionSteps.spec.js

```diff
@@ -73,7 +85,7 @@ Given(
 	}
 );
 
-Given("a board exists in course {string}", (courseName) => {
+Given("a multi-column board exists in course {string}", (courseName) => {
```

Comment:
see here for another example regarding this comment: 
https://github.com/hpi-schul-cloud/e2e-system-tests/pull/284/files#r1931271413

Reply:
same, done!